### PR TITLE
UF-101777: Update municipality and namespace in paths

### PR DESCRIPTION
**Pull request by cheezi0747**

### CODE REVIEW

File: src/integration-test/java/se/sundsvall/supportmanagement/apptest/ErrandCommunicationIT.java [hunk 1]
```diff
@@ -25,8 +25,10 @@
 @DirtiesContext
 class ErrandCommunicationIT extends AbstractAppTest {
 
-	private static final String PATH = "/NAMESPACE.1/2281/errands/"; // 2281 is the municipalityId of Sundsvalls kommun
+	private static final String PATH = "/2281/NAMESPACE.1/errands/"; // 2281 is the municipalityId of Sundsvalls kommun
```
Comment (AndreSoderlund) [inline src/integration-test/java/se/sundsvall/supportmanagement/apptest/ErrandCommunicationIT.java:28]: Remove comment

File: src/main/java/se/sundsvall/supportmanagement/api/EmailIntegrationConfigResource.java [hunk 2]
```diff
@@ -25,54 +29,52 @@
...
-import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
-import se.sundsvall.supportmanagement.api.model.config.EmailIntegration;
-import se.sundsvall.supportmanagement.service.config.EmailIntegrationConfigService;
 
 @RestController
 @Validated
-@RequestMapping("/{namespace}/{municipalityId}/emailIntegrationConfig")
+@RequestMapping("/{municipalityId}/{namespace}/emailIntegrationConfig")
 @Tag(name = "Configuration for email integration", description = "Settings for email integration per namespace/municipality")
 public class EmailIntegrationConfigResource {
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/supportmanagement/api/EmailIntegrationConfigResource.java:48]: Shouldn't be public, same with constructor.

File: src/main/java/se/sundsvall/supportmanagement/integration/db/model/AttachmentEntity.java [hunk 3]
```diff
@@ -29,7 +29,9 @@
 @Entity
 @Table(name = "attachment",
 	indexes = {
-		@Index(name = "idx_attachment_file_name", columnList = "file_name")
+		@Index(name = "idx_attachment_file_name", columnList = "file_name"),
+		@Index(name = "idx_attachment_municipality_id", columnList = "municipality_id"),
+		@Index(name = "idx_attachment_namespace", columnList = "namespace")
 	},
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/supportmanagement/integration/db/model/AttachmentEntity.java:35]: Pretty sure none of these are being used.

File: src/main/java/se/sundsvall/supportmanagement/integration/db/model/CommunicationAttachmentEntity.java [hunk 4]
```diff
@@ -19,13 +20,22 @@
 @Entity
 @Table(
 	name = "communication_attachment",
+	indexes = {
+		@Index(name = "idx_communication_attachment_municipality_id", columnList = "municipality_id"),
+		@Index(name = "idx_communication_attachment_namespace", columnList = "namespace")},
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/supportmanagement/integration/db/model/CommunicationAttachmentEntity.java:25]: Are these useful?

File: src/main/java/se/sundsvall/supportmanagement/integration/db/model/AttachmentEntity.java (hunk 3 above)
Comment (cheezi0747) [reply src/main/java/se/sundsvall/supportmanagement/integration/db/model/AttachmentEntity.java:35]: src/main/java/se/sundsvall/supportmanagement/integration/db/AttachmentRepository.java

**Pull request merged**